index.js

```diff
@@ -32,6 +35,40 @@ Object.defineProperty(LiveReloadPlugin.prototype, 'isRunning', {
   get: function() { return !!this.server; }
 });
 
+String.prototype.hashCode = function() {
```

Comment:
I'd prefer to avoid modifying global prototypes. Naming this `function hashCode(str)` and passing strings to it could be better.

Also, we're already using `crypto`. We could defer to use the builtin.

```js
const hash = crypto.createHash('sha256');
hash.update(str);
return hash.digest('hex');
```

Reply:
Yes sorry. I thought that you would say that. ;-) 

Did not notice that there is already crypto sorry. Switched the function to use crypto.